python/bigipconfigdriver.py

```diff
@@ -554,17 +554,16 @@ def handle_backoff(self):
         if (self._interval and self._interval.is_running() is
                 True):
             self._interval.stop()
-        self.retry_backoff(self.notify_reset)
+        self.retry_backoff()
 
-    def retry_backoff(self, func):
+    def retry_backoff(self):
         """Add a backoff timer to retry in case of failure."""
-        e = threading.Event()
+        timer = threading.Timer(self._backoff_time, self.notify_reset)
```

Comment:
Since this slightly changes behavior I'd be interested in an additional test for that.

1) Start backoff with error (can start it with a high-ish wait)
2) Fix config and trigger a notify
3) Verify timer is cancelled (if timer.finished)